File: Components/WinForms/CS/ChartsMainDemo/Modules/ChartElements/VerticalPanes.cs
using DevExpress.Utils.Menu;

namespace DevExpress.XtraCharts.Demos.ChartElements {
    public partial class VerticalPanesDemo : ChartDemoBase2D {
        public override ChartControl ChartControl { get { return chart; } }

        public VerticalPanesDemo() {
            InitializeComponent();
        }

        protected override DXPopupMenu ConstructPopupMenu(object obj, ChartControl chartControl) {
            return DXMenuHelper.ConstructPaneMenu(obj, chartControl);
        }
    }
}

File: Components/WinForms/VB/ChartsMainDemo/Modules/ChartElements/VerticalPanes.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Utils.Menu

Namespace DevExpress.XtraCharts.Demos.ChartElements
	Partial Public Class VerticalPanesDemo
		Inherits ChartDemoBase2D
		Public Overrides ReadOnly Property ChartControl() As ChartControl
			Get
				Return chart
			End Get
		End Property

		Public Sub New()
			InitializeComponent()
		End Sub

		Protected Overrides Function ConstructPopupMenu(ByVal obj As Object, ByVal chartControl As ChartControl) As DXPopupMenu
			Return DXMenuHelper.ConstructPaneMenu(obj, chartControl)
		End Function
	End Class
End Namespace

